src/components/DebuggingTool/Request.tsx

```diff
@@ -0,0 +1,285 @@
+import {
+  Box,
+  Grid,
+  Tab,
+  Tabs,
+  TextField,
+  FormControl,
+  InputLabel,
+  Button,
+  Select,
+  MenuItem,
+  Divider,
+  ListSubheader,
+} from "@material-ui/core"
+
+import React, { useEffect, useState } from "react"
+import styled from "@emotion/styled"
+import { Collapse } from "react-bootstrap"
+import { Add, Close } from "@material-ui/icons"
+import { nanoid } from "nanoid"
+import { formatDebugRequest } from "../../helpers/format"
+import _ from "lodash"
+
+const TextInput = styled(TextField)`
+  width: 100%;
+`
+
+const FileInput = styled.input`
+  display: none;
+`
+
+function TabPanel(props: any) {
+  const { children, value, index, ...other } = props
+  return (
+    <div
+      role="tabpanel"
```

Comment:
shouldn't we use camelCase here?

Reply:
This is an `aria` convention and not defined by us